### PR TITLE
Yakunara Mug Cup mo Specials Mapping S0

### PR DESCRIPTION
**Pull request by krbrs**
Season0 Override for Episode Numbers (TVDB S0E1 = ANIDB S2, S1 doesn't exist on TVDB). Hopefully this worked. i reversed my previous commits and made sure the fork is updated from upstream this time. sorry for this mess >.>

### CODE REVIEW

File: anime-list-master.xml [hunk 1]
```diff
@@ -43018,6 +43018,9 @@
   </anime>
   <anime anidbid="15660" tvdbid="398106" defaulttvdbseason="1" episodeoffset="" tmdbid="" imdbid="">
     <name>Yakunara Mug Cup mo</name>
+    <mapping-list>
+      <mapping anidbseason="0" tvdbseason="0" start="" end="" offset="-1"></mapping>
```
Comment (reconman) [inline anime-list-master.xml:43022]: start and end are not needed

**Review by reconman — CHANGES_REQUESTED** [review]

File: anime-list-master.xml (hunk 1 above)
Comment (krbrs) [reply anime-list-master.xml:43022]: ok i removed it, i wasn't sure if this is needed even when empty

**Comment by krbrs** [comment]
Should be ready to merge (if specials are always -1 off that is)

**Review by reconman — APPROVED** [review]

**Comment by reconman** [comment]
Sorry, forgot about the PR

**Comment by BrutuZ** [comment]
Don't you need `start="1" end="3"` as well for `offset="-1"` to work?
Just figured that out the hard way with #59, `offset` alone wouldn't have effect on the mapping with ASS+HAMA

**Comment by krbrs** [comment]
I can confirm, that when using the anidb3 override with ASS for plex, the specials do not respect offset="-1". 
ANIDB "S2" is mapped to "S00E02" instead of "S00E01" and so on.

Is it possible to just add "start" without "end" since we don't know how many specials are coming?
Or should an issue be opened on the ASS repository for this?

**Comment by reconman** [comment]
The README says that offset only affects the episodes between start and end, although my instinct was that you don't need it.

Removing the need for `start` and `end` is probably not possible. If you have a TVDB special episode and there are 2 AniDB entries, both only with an offset, then which AniDB entry do you use?

For now I would suggest mapping episode 1 to 0 manually using `;1-0;` and then either also specifying `2-1;3-2;4-3;` or using start="2" and end="4" in a second mapping.

**Comment by krbrs** [comment]
OK, I will do changes to the mug cup mapping and issue another pull request. thanks for the explanation, it makes sense! :)

**Comment by BrutuZ** [comment]
While at it...
```diff
@@ 30513,6 @@
  <anime anidbid="11135" tvdbid="289763" defaulttvdbseason="1" episodeoffset="13" tmdbid="" imdbid="">
    <name>The Idolmaster: Cinderella Girls (2015)</name>
    <mapping-list>
      <mapping anidbseason="0" tvdbseason="0" offset="1"/>
    </mapping-list>
  </anime>
```

**Pull request merged**